### PR TITLE
Remove unused devDependencies

### PR DESCRIPTION
**Pull request by benedikt-richter**
### Summary of changes

* Unused dependencies are removed
* the package is renamed to only consist of valid characters

### CODE REVIEW

**Review by nicarl — APPROVED** [review]

**Review by nicarl — CHANGES_REQUESTED** [review]
Can you also modify the release github action?

**Comment by benedikt-richter** [comment]
The github action does not have to be modified as we rename the files when shipping. These internals are encapsulated in the ship scripts :)

**Review by nicarl — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "OpossumUI",
+  "name": "opossum-ui",
```
Comment (antonbauhofer) [inline package.json:2]: This changes the shown name of the app (i.e. for Mac, the name shown in the top left of the screen and in the applications bar at the bottom). I am not sure about the other systems, but in Mac it is common, to have the name shown including capitalization.
I think I would rather have the warning than changing the name of the app. Or is there another fix for this?

**Review by antonbauhofer — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (benedikt-richter) [reply package.json:2]: Good catch! I agree that we want to show the name "OpossumUI". The config for the builder is adapted to make sure that's the case even though we use a different name here.

**Review by antonbauhofer — APPROVED** [review]
Looks good now.

**Pull request merged**